ui/src/redux/main.ts

```diff
@@ -130,54 +215,48 @@ export const mainSlice = createSlice({
         setExpired: (state, action: PayloadAction<boolean>) => {
             state.expired = action.payload
         },
+        /**
+         * Handle all deployment events that the user can access.
+         * Note that some deployments are triggered by others.
+         */
         handleDeploymentEvent: (state, action: PayloadAction<Event>) => {
-            const user = state.user
-            if (!user) {
-                throw new Error("Unauthorized user.")
-            }
-
             const event = action.payload
+            if (event.kind !== EventKindEnum.Deployment) {
+                return
+            } 
 
-            // Handling the event when the owner is same.
-            if (event.deployment?.deployer?.id !== user.id) {
```

Comment:
Remove the condition to display all deployments even though others are triggered.